packages/components/fields/time-field/docs/description.md

```diff
@@ -0,0 +1 @@
+A controlled date input component for single date.
```

Comment:
I think one line description can be added to the component's `package.json` since the script checks that and if present it will just that one.

Reply:
true, but I prefer it here tbh

Reply:
Having them on the `package.json` has the small advantage of displaying them on the NPMS search (and I assume also on other NPM searching interfaces)

![image](https://user-images.githubusercontent.com/2941328/92373811-2cf80580-f0ff-11ea-8d08-b9bfd964c407.png)

Reply:
Yes, I would also prefer to ALWAYS have some kind of description in the `package.json`.'

The `description.md` file was only meant to be used in case the description section needed more information and MD syntax.
Nevertheless, a summary/short sentence should be also provided in the `package.json`.

Reply:
will do as last commit